README.md

```diff
@@ -4,270 +4,48 @@
-- Temporary Files and Directories - [tempfile](https://crates.io/crates/tempfile)
-
-## Docker
-An environment that includes a local [MinIO](https://min.io/) instance and an edge node using the [MinIO](https://min.io/)
-instance as the remote object store, can be set up using [Docker](https://docs.docker.com/). Note that since
-[Rust](https://www.rust-lang.org/) is a compiled language and a more dynamic `modelardbd` configuration might be needed,
-it is not recommended to use the [Docker](https://docs.docker.com/) environment during active development of `modelardbd`.
-It is however ideal to use during more static deployments or when developing components that utilize `modelardbd`.
-
-Downloading [Docker Desktop](https://docs.docker.com/desktop/) is recommended to make maintenance of the created
-containers easier. Once [Docker](https://docs.docker.com/) is set up, the [MinIO](https://min.io/) instance can be
-created by running the services defined in [docker-compose-minio.yml](docker-compose-minio.yml). The services can
-be built and started using the command:
-
-```shell
-docker-compose -p modelardata-minio -f docker-compose-minio.yml up
-```
-
-After the [MinIO](https://min.io/) service is created, a [MinIO](https://min.io/) client is created to initialize
-the development bucket `modelardata`, if it does not already exist. [MinIO](https://min.io/) can be administered through
-its [web interface](http://localhost:9001). The default username and password, `minioadmin`, can be used to log in.
-A separate compose file is used for [MinIO](https://min.io/) so an existing [MinIO](https://min.io/) instance can be
-used when `modelardbd` is deployed using [Docker](https://docker.com/), if necessary.
-
-Similarly, the `modelardbd` instance can be built and started using the command:
-
-```shell
-docker-compose -p modelardbd up
-```
-
-The instance can then be accessed using the Apache Arrow Flight interface at `grpc://127.0.0.1:9999`.
-
-## Contributions
-Contributions to all aspects of ModelarDB are highly appreciated and do not need
-to be in the form of code. For example, contributions can be:
-
-- Helping other users.
-- Writing documentation.
-- Testing features and reporting bugs.
-- Writing unit tests and integration tests.
-- Fixing bugs in existing functionality.
-- Refactoring existing functionality.
-- Implementing new functionality.
-
-Any questions or discussions regarding a possible contribution should be posted
-in the appropriate GitHub issue if one exists, e.g., the bug report if it is a
-bugfix, and as a new GitHub issue otherwise.
+Refer to the [Development](docs/dev/README.md) section of the documentation for an overview of the structure of the 
+project, a detailed description of each major component, and an outline of the guidelines that should be adhered 
+to when contributing to the project.
+
+## Research-Based
+The prototype of ModelarDB was implemented using [Apache Spark](https://www.h2database.com/html/main.html),
```

Comment:
I think there are some benefits to mentioning what was used in previous versions, since it helps in showing what we have changed to improve the implementation. As such I think it might be best to keep the text as it is. What do you think @skejserjensen?

Reply:
I think it is beneficial that we state that the original prototype was JVM-based and built at AAU so people can see that the general ideas are thoroughly tested, but we should also make it very explicit that the JVM-based version and the Rust-based version are completely different systems so people do not think the Rust-based version is research quality code. I think the current level of information is fine, but that it should be more explicit which version we talk about, currently "this" is used when talking about both. Maybe we can use something like "the deprecated prototype" to talk about the old JVM-based version and  "the current version" to talk about the Rust-based version?

Reply:
Fixed.

Reply:
Sorry to pedantic here, but I do still think it is unnecessary to go into details about the prototype here and mention Cassandra, Spark, H2... If someone is interested in prototype, they could follow the link. Now, I think the text may mislead people into believing that we still use H2, Spark and Cassandra. The text basically says that the old version used them and that it was JVM-based, and that the current version is Rust-based (but not that it does not rely on H2, Spark, and Cassandra anymore...)

Reply:
Removed details about the deprecated prototype.